packages/hyperion-flowlet/src/Index.ts

```diff
@@ -20,115 +20,115 @@ export function initFlowletTrackers(flowletManager: FlowletManager) {
-    // IGlobalEventHandlers.onmousedown,
-    // IGlobalEventHandlers.onmouseenter,
-    // IGlobalEventHandlers.onmouseleave,
-    // IGlobalEventHandlers.onmousemove,
-    // IGlobalEventHandlers.onmouseout,
-    // IGlobalEventHandlers.onmouseover,
-    // IGlobalEventHandlers.onmouseup,
-    // IGlobalEventHandlers.onpause,
-    // IGlobalEventHandlers.onplay,
-    // IGlobalEventHandlers.onplaying,
-    // IGlobalEventHandlers.onpointercancel,
-    // IGlobalEventHandlers.onpointerdown,
-    // IGlobalEventHandlers.onpointerenter,
-    // IGlobalEventHandlers.onpointerleave,
-    // IGlobalEventHandlers.onpointermove,
-    // IGlobalEventHandlers.onpointerout,
-    // IGlobalEventHandlers.onpointerover,
-    // IGlobalEventHandlers.onpointerup,
-    // IGlobalEventHandlers.onprogress,
-    // IGlobalEventHandlers.onratechange,
-    // IGlobalEventHandlers.onreset,
-    // IGlobalEventHandlers.onresize,
-    // IGlobalEventHandlers.onscroll,
-    // IGlobalEventHandlers.onsecuritypolicyviolation,
-    // IGlobalEventHandlers.onseeked,
-    // IGlobalEventHandlers.onseeking,
-    // IGlobalEventHandlers.onselect,
-    // IGlobalEventHandlers.onselectionchange,
-    // IGlobalEventHandlers.onselectstart,
-    // IGlobalEventHandlers.onslotchange,
-    // IGlobalEventHandlers.onstalled,
-    // IGlobalEventHandlers.onsubmit,
-    // IGlobalEventHandlers.onsuspend,
-    // IGlobalEventHandlers.ontimeupdate,
-    // IGlobalEventHandlers.ontoggle,
-    // IGlobalEventHandlers.ontouchcancel,
-    // IGlobalEventHandlers.ontouchend,
-    // IGlobalEventHandlers.ontouchmove,
-    // IGlobalEventHandlers.ontouchstart,
-    // IGlobalEventHandlers.ontransitioncancel,
-    // IGlobalEventHandlers.ontransitionend,
-    // IGlobalEventHandlers.ontransitionrun,
-    // IGlobalEventHandlers.ontransitionstart,
-    // IGlobalEventHandlers.onvolumechange,
-    // IGlobalEventHandlers.onwaiting,
-    // IGlobalEventHandlers.onwebkitanimationend,
-    // IGlobalEventHandlers.onwebkitanimationiteration,
-    // IGlobalEventHandlers.onwebkitanimationstart,
-    // IGlobalEventHandlers.onwebkittransitionend,
-    // IGlobalEventHandlers.onwheel,
-    // IGlobalEventHandlers.onafterprint,
-    // IGlobalEventHandlers.onbeforeprint,
-    // IGlobalEventHandlers.onbeforeunload,
-    // IGlobalEventHandlers.ongamepadconnected,
-    // IGlobalEventHandlers.ongamepaddisconnected,
-    // IGlobalEventHandlers.onhashchange,
-    // IGlobalEventHandlers.onlanguagechange,
-    // IGlobalEventHandlers.onmessage,
-    // IGlobalEventHandlers.onmessageerror,
-    // IGlobalEventHandlers.onoffline,
-    // IGlobalEventHandlers.ononline,
-    // IGlobalEventHandlers.onpagehide,
-    // IGlobalEventHandlers.onpageshow,
-    // IGlobalEventHandlers.onpopstate,
-    // IGlobalEventHandlers.onrejectionhandled,
-    // IGlobalEventHandlers.onstorage,
-    // IGlobalEventHandlers.onunhandledrejection,
-    // IGlobalEventHandlers.onunload,
+    IWindow.ondevicemotion,
```

Comment:
I realized we are also not handling these cases for flowlet wrapping.

Reply:
nice find